mbview.js

```diff
@@ -10,10 +11,10 @@ const objectAssign = require('object-assign');
 
 app.set('views', __dirname + '/views');
 app.set('view engine', 'ejs');
-app.use(express.static('public'));
+app.use(express.static(__dirname + '/public'));
```

Comment:
any idea where it pulls public from without setting _dirname + '/... ?

I just opened https://github.com/mapbox/mbview/pull/96 which would conflict with this change, but without making the change mbview was broken for me.

Reply:
My guess would be your current working directory.  
<details><summary>I couldn't find any docs with a definite answer.</summary>

None of
- https://expressjs.com/en/starter/static-files.html
- https://expressjs.com/en/4x/api.html#express.static
- https://expressjs.com/en/resources/middleware/serve-static.html
have an answer that I could find.

</details>

Reply:
I couldn't find a single directory with that name, even in node_modules.
```sh
# in root of repo
docker run -it $(pwd):/root node:12-alpine sh
cd && rm -rf ./node_modules && npm ci && find . -type d -name public
echo $? && exit 
```